dask_expr/tests/test_groupby.py

```diff
@@ -381,6 +381,19 @@ def test_groupby_median(df, pdf):
     assert_eq(df.groupby("x").median()["y"], pdf.groupby("x").median()["y"])
 
 
+def test_groupby_ffill_bfill(pdf):
+    pdf["y"] = pdf["y"].astype("float64")
+    pdf.loc[pdf.index % 2 == 0, "y"] = np.nan
+    df = from_pandas(pdf, npartitions=10)
```

Comment:
y is either always NaN or always non-NaN within each group, which makes ffill/bfill a no-op.
This passes:
```python
    assert_eq(pdf.groupby("x").ffill().sort_index(), pdf[["y", "z"]])
    assert_eq(pdf.groupby("x").bfill().sort_index(), pdf[["y", "z"]])
```

I think it's also important that the test ensures that we're testing the use case of cross-partition ffill and bfill.

We should also make sure that a change in the pdf fixture in the future won't accidentally neuter the unit test by making these two use cases disappear.